grid_map_msgs/msg/GridMapInfo.msg

```diff
@@ -1,5 +1,5 @@
 # Header (time and frame)
-Header header
+std_msgs/Header header
```

Comment:
In consideration of #231 could you move the header into `GridMap.msg`. This will trigger some problems down the line, but a shortcoming of the ROS1 version will be solved.

Reply:
resolved in d12ab7f